nav2_util/src/lifecycle_utils.cpp

```
@@ -98,4 +99,65 @@ void reset_lifecycle_nodes(
   }
 }
 
+static void processErrorLifecycleNode(
+  const std::string & node_name,
+  const std::chrono::seconds service_call_timeout,
+  const int retries)
+{
+  LifecycleServiceClient sc(node_name);
+  uint8_t transition;
+  // Each state of the lifecycle node has a specific transition
+  // to the ErrorProcessing state. Some states do not support transitions
+  // to ErrorProcessing.
+  switch (sc.get_state(service_call_timeout)) {
+    case State::PRIMARY_STATE_UNKNOWN:
```

For all of these return cases, please throw an exception since they're invalid.